### PR TITLE
Integrate with Github Actions

### PR DESCRIPTION
**Pull request by vadeg**
* Add Postgres container and run integration tests
* Add caching for maven dependencies
* Fix one integration test to work in different timezones

Integration tests are disabled for the moment. Many tests are failing because of incorrect tests with date times. (see #75 )

Fixes #74

### CODE REVIEW

File: .github/workflows/workflow.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    services:
+      postgres:
+        image: postgres:9.6.10
```
Comment (Jan-M) [inline .github/workflows/workflow.yml:18]: lets get rid of those 9.6.10 DBs :D

Comment (vadeg) [reply .github/workflows/workflow.yml]: Updated

**Comment by vadeg** [comment]
👍

**Comment by Jan-M** [comment]
:+1:

**Pull request merged**